app/component/menu/menu-list/menu-list.html

```diff
@@ -0,0 +1,7 @@
+<section class="menu-container">
+  <h4 class="menu-title">menu</h4>
+  <div class="menu-pic-area">
+    <img ng-repeat="pic in menuListCtrl.pics" class="menu-pic" src="{{pic}}"></img>
+  </div>
+  <button class="btn-std" ng-click="menuListCtrl.updatePics(menuListCtrl.biz)">MAKE SOMETHING HAPPEN</button>
```

Comment:
Since you used a binding for biz, you can remove the param from updatePics, and reference this.biz inside that method. See comment below.